marlowe-runtime-web/test/Spec.hs

```diff
@@ -43,6 +46,193 @@ main = hspec do
 
 openAPISpec :: Spec
 openAPISpec = do
+  focus do
+    it "finds no problems with empty schema" do
```

Comment:
I like that you have unit tests for the lining itself, however the lining tool is ultimately just a test for the open API documentation we generate, so I'd like to see a test added that is similar to this one, but for the open API documentation for our API.

Reply:
Yes, that makes sense. I'll add it in.